_posts/0000-01-02-helfrae.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
It says that I need to chance this line. I just took a Beginners course for GitHub and learning how to use it